### PR TITLE
brand.uiowa.edu site split export as v3

### PR DESCRIPTION
**Pull request by joewhitsitt**
relates to https://github.com/uiowa/uiowa/issues/5203

The idea behind this is to allow brand.uiowa.edu to continue as a v2 site in PROD while a v3 site is in stage while maintaining their custom functionality. We cannot have import errors or loss of functionality whether main is deployed to stage or prod.

## To Test

- checkout brand_v3
- for good measure `ddev composer install && ddev blt frontend`
- If you have the brand site local, `ddev drush @brand.local sql-drop && ddev drush @brand.local cr`
- First try and sync the PROD site to see if there are any import issues: `ddev blt ds --site=brand.uiowa.edu`
- As a webmaster, poke around the site making sure you can't see layout builder or page replicate functionality but can see the lockup functionality.
- Run `ddev drush @brand.local cr` and then try to export the site `ddev drush @brand.local config-split:export site` and review, if any, differences.
  - Disregard and remove the following (`git clean -fd config/sites/brand.uiowa.edu/`):
    - config/sites/brand.uiowa.edu/system.action.user_add_role_action.webmaster.yml
    - config/sites/brand.uiowa.edu/system.action.user_remove_role_action.webmaster.yml
    - config/sites/brand.uiowa.edu/user.role.webmaster.yml
- Next try to install brand as a fresh v3 site `ddev drush @brand.local sql-drop && ddev drush @brand.local cr && ddev blt drupal:install --site=brand.uiowa.edu` enable some modules `ddev drush @brand.local en mailsystem swiftmailer workbench_email brand_core` and then run the site config import `ddev drush @brand.local config:import --source ../config/sites/brand.uiowa.edu --partial` followed by `ddev drush @brand.local cim` and look for any import errors.
- Poke around the site as a webmaster (make an account with webmaster role to masquerade with) and make sure you can do layout builder type stuff along with the lockup related stuff.
- Run `ddev drush @brand.local cr` and then try to export the site `ddev drush @brand.local config-split:export site` and review, if any, differences.
  - Disregard and remove the following (`git clean -fd config/sites/brand.uiowa.edu/`):
    - config/sites/brand.uiowa.edu/system.action.user_add_role_action.webmaster.yml
    - config/sites/brand.uiowa.edu/system.action.user_remove_role_action.webmaster.yml
    - config/sites/brand.uiowa.edu/user.role.webmaster.yml

### CODE REVIEW

File: config/sites/brand.uiowa.edu/config_split.patch.administerusersbyrole.settings.yml [hunk 1]
```diff
@@ -1,4 +1,4 @@
 adding:
   roles:
-    collaborator: safe
+    collaborator: unsafe
```
Comment (joewhitsitt) [inline config/sites/brand.uiowa.edu/config_split.patch.administerusersbyrole.settings.yml]: @richardbporter does this look correct to you?

**Comment by joewhitsitt** [comment]
looks like i lost collaborator uiowa_auth settings. i will work on getting that back for now. should that be tracked in config anymore or should that be ignored?

**Comment by richardbporter** [comment]
> looks like i lost collaborator uiowa_auth settings. i will work on getting that back for now. should that be tracked in config anymore or should that be ignored?

It should be tracked.

File: config/sites/brand.uiowa.edu/config_split.patch.administerusersbyrole.settings.yml (hunk 1 above)
Comment (richardbporter) [reply config/sites/brand.uiowa.edu/config_split.patch.administerusersbyrole.settings.yml]: I don't think so based on what is in config/default.

**Comment by bspeare** [comment]
@joewhitsitt I ran `ddev drush @brand.local config-split:export site` and didn't see any of those files you mentioned in the fourth bullet point.

**Comment by joewhitsitt** [comment]
>didn't see any of those files you mentioned in the fourth bullet point.

Nice! Those were gremlins on my local I couldn't explain, nor justify adding to the split.

**Comment by bspeare** [comment]
>Poke around the site as a webmaster (make an account with webmaster role to masquerade with) and make sure you can do layout builder type stuff along with the lockup related stuff.

I'm not seeing layout or replicate options as a webmaster.

**Comment by bspeare** [comment]
OK, trying this again.  These were the files deleted on step 4 - is this correct?

```
	deleted:    config/sites/brand.uiowa.edu/system.action.user_remove_role_action.collaborator.yml
	deleted:    config/sites/brand.uiowa.edu/taxonomy.vocabulary.orgs.yml
	deleted:    config/sites/brand.uiowa.edu/user.role.collaborator.yml
	deleted:    config/sites/brand.uiowa.edu/views.view.lockup_moderation.yml
	deleted:    config/sites/brand.uiowa.edu/views.view.lockups.yml
	deleted:    config/sites/brand.uiowa.edu/workbench_email.workbench_email_template.approved_author.yml
	deleted:    config/sites/brand.uiowa.edu/workflows.workflow.lockup.yml
```

**Comment by bspeare** [comment]
I tried this again and still have the same issue https://github.com/uiowa/uiowa/pull/5223#issuecomment-1117552866.  Maybe my local version is having a caching problem?  I think we need someone else to test this to see if they get the same results.

**Comment by joewhitsitt** [comment]
https://github.com/uiowa/uiowa/pull/5223#issuecomment-1117616774 makes me think something is up with the steps or your local because this is essentially all of the site split's complete config.

**Comment by joewhitsitt** [comment]
I'd be willing to huddle with anyone who tries this to see what is up

**Comment by bspeare** [comment]
I just had to restart my ddev, so there was probably something going on with my local environment.   I'll try this again in a bit.

**Comment by bspeare** [comment]
I tried this again after restarting `ddev`.  Must be local, user error or something. 

```
	deleted:    config/sites/brand.uiowa.edu/system.action.user_add_role_action.collaborator.yml
	deleted:    config/sites/brand.uiowa.edu/system.action.user_remove_role_action.collaborator.yml
	deleted:    config/sites/brand.uiowa.edu/taxonomy.vocabulary.orgs.yml
	deleted:    config/sites/brand.uiowa.edu/user.role.collaborator.yml
	deleted:    config/sites/brand.uiowa.edu/views.view.lockup_moderation.yml
	deleted:    config/sites/brand.uiowa.edu/views.view.lockups.yml
	deleted:    config/sites/brand.uiowa.edu/workbench_email.workbench_email_template.approved_author.yml
	deleted:    config/sites/brand.uiowa.edu/workflows.workflow.lockup.yml
```

**Comment by joewhitsitt** [comment]
Hmm. Okay. I am going to pull this down again and look

**Comment by pyrello** [comment]
I'm testing this.

**Comment by joewhitsitt** [comment]
Pumping the breaks to discuss blank slate, coronavirus method and p2lb on Monday

**Pull request closed without merging**